spec/models/service_template_transformation_plan_spec.rb

```diff
@@ -37,6 +37,18 @@
     }
   end
 
+  describe '.public_service_templates' do
+    it 'display public service templates' do
+      st1 = FactoryGirl.create(:service_template_transformation_plan)
+      st2 = FactoryGirl.create(:service_template)
+
+      expect(st1.internal).to be_truthy
+      expect(st2.internal).to be_falsey
+      expect(ServiceTemplate.public_service_templates.count).to eq(1)
```

Comment:
These two expectations can be merged into:
```ruby
expect(ServiceTemplate.public_service_templates).to match_array([st2])
```

Reply:
Changed.